moped-database/migrations/1658526083849_insert_into_public_moped_components/up.sql

```diff
@@ -0,0 +1,10 @@
+INSERT INTO "public"."moped_components"(
+	"component_name", "status_id", "component_subtype", "line_representation") 
+VALUES 
+	(E'Lighting', 1, E'Street lighting', true),
+	(E'Lighting', 1, E'Intersection lighting', false),
+	(E'Transit', 1, E'Bus stop', false);
+
+UPDATE "public"."moped_components"
+SET line_representation = true
+WHERE component_id = 0;
```

Comment:
@mddilley here's where the Project Extent component is changing to line representation.

I'm pretty sure Chia checked, but I'll double check to see if we have any existing project extent components stored as points in the DB. We'd need to convert those to lines